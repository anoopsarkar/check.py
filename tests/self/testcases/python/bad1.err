running exprs div
running exprs minus
running exprs mod
running exprs mult
running exprs multi
running exprs plus
running exprs toomanytokens
running exprs unknownop
running nl 1
running nl 2
running nl 3
running nl 4
running replacechar 1
running replacechar 2
running replacechar 3
running sortnums 1
running sortnums 2
running sortnums 3
running sortnums 4
running sortnums 5
