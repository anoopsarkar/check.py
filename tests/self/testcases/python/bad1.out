exprs div : failed : failure exit value, stdout
exprs minus : passed
exprs mod : passed
exprs mult : passed
exprs multi : failed : failure exit value, stdout
exprs plus : failed : failure exit value, stdout
exprs toomanytokens : passed
exprs unknownop : passed
nl 1 : failed : failure exit value, stdout
nl 2 : passed
nl 3 : passed
nl 4 : passed
replacechar 1 : passed
replacechar 2 : failed : stdout
replacechar 3 : failed : failure exit value, stdout
sortnums 1 : passed
sortnums 2 : passed
sortnums 3 : passed
sortnums 4 : passed
sortnums 5 : failed : failure exit value, stdout
exprs : 5 / 8
sortnums : 4 / 5
nl : 3 / 4
replacechar : 1 / 3
total : 13 / 20
